protos/model/result.proto

```diff
@@ -169,6 +170,25 @@ message CharacterBuckets {
   repeated DescriptiveStats buckets = 1 [json_name="buckets"];
 }
 
+message TargetBucketStats {
+  uint32 bucket_size = 1 [json_name="bucket_size"];
+  map<int32, TargetBuckets> targets = 2 [json_name="targets"];
+}
+
+message TargetBuckets {
+  TargetBucket overall = 1 [json_name="overall"];
+  TargetBucket target = 2 [json_name="target"];
+}
```

Comment:
nit: may be cleaner if we can move `overall` to `TargetBucketStats` and do the cross-target sum in agg flush rather than deferring it to the UI